base/test/filewritermodule_tests.cpp

```diff
@@ -9,7 +9,7 @@
 #include "Logger.h"
 #include "AIPExceptions.h"
 #include "test_utils.h"
-
+#include "fstream"
```

Comment:
it should be #include <fstream>  quotes are used for custom code while angular brackets are used for standard header